docs/content/en/docs/getting-started/production-environment/baremetal-getstarted.md

```diff
@@ -195,7 +195,8 @@ Follow these steps if you want to use your initial cluster to create and manage
      >   eksctl anywhere generate hardware -z updated-hardware.csv > updated-hardware.yaml
      >   kubectl apply -f updated-hardware.yaml
      >   ```
-     > * If you want to create multiple workload clusters, you should create them one by one, and make sure its completely up and running successfully by checking tinkerbell machines, before attempting to create the next workload cluster. Attempting to create multiple workload clusters without letting the requests complete successfully can cause hardware race conditions. In a situation where sufficient hardware is not provisioned, the creation would fail at the CAPT level.
+     > * For creating multiple workload clusters, it is essential that the hardware labels and selectors defined for a given workload cluster are unique to that workload cluster. For instance, for an EKSA cluster named `eksa-workload1`, the hardware that is assigned for this cluster should have labels that are only going to be used for this cluster like `type=eksa-workload1-cp` and `type=eksa-workload1-worker`.
```

Comment:
Can we link to https://anywhere.eks.amazonaws.com/docs/reference/clusterspec/baremetal/#hardwareselector